data/releases/eol.yaml

```diff
@@ -1,4 +1,7 @@
 branches:
+  - release: "1.22"
+    finalPatchRelease: "1.22.16"
+    endOfLifeDate: 2022-11-09
```

Comment:
We may have one more 1.22 in December containing infra changes.

/cc @kubernetes/k8s-infra-gcp-org-admins  @kubernetes/release-managers

Reply:
@saschagrunert That's a good point. Do we want to revert this change for now and leave a note that one more 1.22 release might be issued to migrate to `registry.k8s.io`?

Reply:
Should we split this into 2 PRs? One containing the December 7 patch release decision and November patch updates and another with potentially another 1.22 with the infra changes?

This way we have fresh information on the website about the early December patch releases at the least.

Reply:
@palnabarun I thought about that as well. I'll take care of that.

Reply:
I dropped changes related to 1.22 from this PR. PTAL @palnabarun @saschagrunert

Reply:
I created #37897 for 1.22.17. Let's discuss this there.